lib/smart_answer_flows/marriage-abroad.rb

```diff
@@ -219,7 +219,7 @@ def define
 
       outcome :outcome_opposite_sex_marriage_indonesia
 
-      outcome :outcome_os_laos
+      outcome :outcome_opposite_sex_marriage_laos_with_lao_national
```

Comment:
I wonder whether `outcome_opposite_sex_marriage_in_laos_with_lao_national` might be more readable.